templates/experiments/share/widget.html

```diff
@@ -8,4 +8,4 @@
 <!-- place this in the <head> section -->
 \<script type="module" src="https://unpkg.com/open-chat-studio-widget@0.2.0/dist/open-chat-studio-widget/open-chat-studio-widget.esm.js"\>\<\/script\>
     <!-- place this where you want the widget button to appear -->
-    <open-chat-studio-widget visible="false"  bot-url="{% absolute_url embed_url %}" button-text="Let's Chat" position="right" expanded="false"></open-chat-studio-widget>
+    <open-chat-studio-widget visible="false"  bot-url="{% absolute_url embed_url %}" button-text="Let's Chat" position="right" expanded="false" icon-url="{% absolute_url '/static/images/favicons/favicon.svg' %}"></open-chat-studio-widget>
```

Comment:
Was this change intentional? I don't think it makes sense until we've released a new version.